### PR TITLE
Add ViewBuilder to TimelineStep init

### PR DESCRIPTION
**Pull request by Parabak**
Closes issue#195

### CODE REVIEW

File: Sources/Orbit/Support/Components/TimelineStepText.swift [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import SwiftUI
+
+/// Text content of a TimelineStep.
+///
+/// - Related components
```
[the remark this answers is not in the document]
Comment (Parabak) [reply Sources/Orbit/Support/Components/TimelineStepText.swift:5]: Oh, thanks! I didn't know that. I've mimic that part from other TimelineStepText

[the remark this answers is not in the document]
Comment (Parabak) [reply Sources/Orbit/Support/Components/TimelineStepText.swift]: Uopdated

File: Sources/Orbit/Support/Components/TimelineStep.swift [hunk 2]
```diff
@@ -67,62 +64,72 @@ public struct TimelineStep: View {
...
+
+public extension TimelineStep where Content == TimelineStepText {
+
+    /// Creates Orbit TimelineStep component with text details.
+    init(_ label: String = "", sublabel: String = "", style: TimelineStepStyle = .default, textContent: String) {
         self.label = label
         self.sublabel = sublabel
         self.style = style
-        self.content = content
+        self.content = TimelineStepText(text: textContent, style: style)
```
Comment (PavelHolec) [inline Sources/Orbit/Support/Components/TimelineStep.swift:83]: It is probably better to call the `self.init()` for reusability

**Comment by PavelHolec** [comment]
Please rename branch to GitHub naming 🙏

**Comment by Parabak** [comment]
@PavelHolec I had to close this MR as you can't change the source branch name. New MR with updates is here https://github.com/kiwicom/orbit-swiftui/pull/234

**Comment by sjavora** [comment]
> GitHub naming

@PavelHolec we have conventions around that? Are they written down somewhere? 🤔

**Pull request closed without merging**